.github/environment/staging.env

```diff
@@ -0,0 +1,5 @@
+AWS_REGION=ap-southeast-2
+CODEARTIFACT_DOMAIN=gamma-aodn-org-au
+CODEARTIFACT_REPO=maven-aodn-store
+ECR_REGISTRY=450356697252.dkr.ecr.ap-southeast-2.amazonaws.com
+ECR_REPOSITORY=es-indexer
```

Comment:
I'm just curious why you don't declare these environments inside the settings tab of the repo but declaring them in files instead?

Reply:
Ah these are just here as a record. It's easy then to see what the values should be and you can also update the actual environment variables using the GH CLI (as per the README)